### PR TITLE
FIX: 'Statistics' tab

### PR DESCRIPTION
**Pull request by pawelzwronek**
FIX: 'Statistics' tab
ADD: planet name
stat

### CODE REVIEW

File: OGame UI++/src/tab-stats.js [hunk 1]
```diff
@@ -35,7 +36,7 @@ var fn = function () {
       var rentabilityTimes = [];
       for (var coords in config.my.planets) {
         var planet = config.my.planets[coords];
-        if (!planet.resources) {
```
Comment (eswak) [inline OGame UI++/src/tab-stats.js]: which case did you fix here ?

**Comment by eswak** [comment]
I prefer the resource display like `+1,027M /j` instead of `+ 1.027.355 /j` though

File: OGame UI++/src/tab-stats.js (hunk 1 above)
Comment (pawelzwronek) [inline OGame UI++/src/tab-stats.js]: It's a bigger problem with multiple users signing on the same browser in. The effect is that the list of planets got merged and now I dont know why but planet.coords (for not my planet) is null and tab with stats doesn't work at all. I think config have to be saved per user.

**Comment by pawelzwronek** [comment]
Maybe this?
stat2

**Comment by pawelzwronek** [comment]
Do you have an idea why some cells have around them this rectangle, like it was selected ?

**Comment by eswak** [comment]
it's the next 5 most rentable buildings (you should build LemAntymon's mines next)

**Comment by eswak** [comment]
can you fix the weird bottom padding that your PR adds ? maybe adjusting `line-height` of the planet's coords/name will do the trick

**Comment by eswak** [comment]
I didn't think about the real-time ticking of the ressources, does it look cooler with the full number (what you originally suggested) instead of the "M" "k" compression ?

**Comment by eswak** [comment]
I'll include your 2 reports in the next release (planet name in stats tab & config per player)

**Comment by eswak** [comment]
See https://github.com/eswak/chrome-ogame-ui-extension/pull/35/commits/cfa1ae5abd255d95bbd2e90cd587f39b839d3b49 for planet name & https://github.com/eswak/chrome-ogame-ui-extension/pull/35/commits/bd3340a87a5f36dc6a2a51ce4d6f04d8cf8de149 for per-player config.

I doubt more than 2 configs will fit in localStorage though so I hope you're not multi-accounting on more than 2 accounts for the same universe

**Comment by pawelzwronek** [comment]
No no, I share my laptop with my friend and ours planets gets mixed in ui++. Why you doubt 2 configs will fit in localStorage? I have plenty of free space on my hdd :), you are not?

**Comment by eswak** [comment]
`localStorage` is limited to a few MB (10MB I think) per origin (`sxxx.ogame.gameforge.com`) and the JSON for the config is quite big because of the historical data of the whole universe, the notes, etc. So if there are multiple entries in the localStorage, an error can occur. I had this problem on my machine when backing up more than once my config : 
> Uncaught DOMException: Failed to execute 'setItem' on 'Storage': Setting the value of 'og-enhancements-backup2' exceeded the quota.

you can achieve this error using the following code in the chrome console :
```js
var playerId = $('[name=ogame-player-id]').attr('content');
localStorage.setItem('og-enhancements-backup1', localStorage.getItem('og-enhancements-' + playerId));
localStorage.setItem('og-enhancements-backup2', localStorage.getItem('og-enhancements-' + playerId));
```
then don't forget to clean :
```js
localStorage.removeItem('og-enhancements-backup1');
localStorage.removeItem('og-enhancements-backup2');
```

**Comment by pawelzwronek** [comment]
Mine is 5MB. Try this: http://jsfiddle.net/53xcc4LL/

**Comment by eswak** [comment]
same for me ^^ of course, we use the same browser (I did the update from v58 to v59 at home after what you told me the other day)

**Pull request closed without merging**